cmd/api/book/service.go

```diff
@@ -57,7 +68,17 @@ func (s *Service) CreateBook(ctx context.Context, req CreateBookRequest) (Book,
 		UpdatedAt: createdAt,
 		//Archived is set to false by defalut inside database
 	}
-	return s.repo.CreateBook(ctx, newBook)
+
+	b, err := s.repo.CreateBook(ctx, newBook)
+	if err == nil {
+		go func() {
+			ctx, cancel := context.WithTimeout(context.Background(), s.notificationsTimeout)
+			defer cancel()
+			err := s.ntf.BookCreated(ctx, req.Name, *req.Inventory)
+			log.Println(err)
```

Comment:
Please test the error before logging.
You should only log if there was an error.
```suggestion
			if err != nil {
			        log.Println(err)
			}

Reply:
Off course! You're right.